liota/dccs/iotcc.py

```diff
@@ -590,3 +625,28 @@ def _unregistration(self, msg_id, ref_entity):
                 "name": ref_entity.name
             }
         }
+
+    def get_properties(self, resource_uuid):
+        """ get list of properties with resource uuid """
+        log.info("Get properties defined with IoTCC for resource {0}".format(resource_uuid))
+        self.prop_list = None
+
+        def on_response(msg):
+            try:
+                log.debug("Received msg: {0}".format(msg))
+                json_msg = json.loads(msg)
+                log.debug("Processed msg: {0}".format(json_msg["type"]))
+                if json_msg["type"] == "get_properties_response" and json_msg["body"]["uuid"] != "null" and \
+                                json_msg["body"]["uuid"] == resource_uuid:
+                    log.info("FOUND PROPERTIE LIST: {0}".format(json_msg["body"]["propertyList"]))
+                    self.prop_list = json_msg["body"]["propertyList"]
+                    log.info("prop_list:{0}".format(self.prop_list))
+                else:
+                    log.info("Waiting for getting properties")
+                    on_response(self.recv_msg_queue.get(True,30))
```

Comment:
We are using wit time of 300 seconds at other places. Why is it 30 seconds here?